packages/lab/src/system-status/SystemStatusActions.css

```diff
@@ -0,0 +1,6 @@
+.saltSystemStatusActions {
+  margin-bottom: auto;
+  display: flex;
+  gap: var(--salt-spacing-100);
+  min-height: var(--salt-size-base);
```

Comment:
Not necessary an issue, i'm wondering why this is needed if button has height?

Reply:
removed. makes sense. will test it further when status buttons are added